sdk/storage/azure-storage-perf/src/main/java/com/azure/storage/file/datalake/perf/core/FileTestBase.java

```diff
@@ -0,0 +1,52 @@
+package com.azure.storage.file.datalake.perf.core;// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+import com.azure.perf.test.core.PerfStressOptions;
+import com.azure.storage.file.datalake.DataLakeFileAsyncClient;
+import com.azure.storage.file.datalake.DataLakeFileClient;
+import com.azure.storage.file.share.ShareFileAsyncClient;
+import com.azure.storage.file.share.ShareFileClient;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.UUID;
+
+public abstract class FileTestBase<TOptions extends PerfStressOptions> extends DirectoryTest<TOptions> {
+
+    public static final int DEFAULT_BUFFER_SIZE = 8192;
+    protected final DataLakeFileClient dataLakeFileClient;
+    protected final DataLakeFileAsyncClient dataLakeFileAsyncClient;
+
+    public FileTestBase(TOptions options) {
+        super(options);
+
+        String fileName = "randomfiletest-" + UUID.randomUUID().toString();
+
+        dataLakeFileClient =  dataLakeDirectoryClient.getFileClient(fileName);
+        dataLakeFileAsyncClient = dataLakeDirectoryAsyncClient.getFileAsyncClient(fileName);
+    }
+
+    @Override
+    public Mono<Void> setupAsync() {
+        return dataLakeFileAsyncClient.create().then(super.cleanupAsync());
+    }
+
+    public long copyStream(InputStream input, OutputStream out) throws IOException {
```

Comment:
I'm seeing this code in multiple places, can we move it into the base framework as a utility method?